lib/listudy_web/templates/study/show.html.eex

```diff
@@ -134,21 +137,26 @@
-  i18n.info_arrows = "<%= dgettext "study", "At first the arrows will show you which moves are in this study. Once you have played these moves twice, the arrows are no longer shown." %>";
+  i18n.info_arrows = "<%= dgettext "study", "By default arrows will provide hints to show which moves are in this study. Using the options below you can control for how many moves the arrows/hints should stay visible. By default they stay visible until all moves in a position has been played twice. Any non-playable arrows from the study itself will appear as transparent arrows." %>";
   i18n.error_wrong_move = "<%= dgettext "study", "This move is not in the study, try again." %>";
   i18n.success_right_move = "<%= dgettext "study", "Right move!" %>";
   i18n.success_end_of_line = "<%= dgettext "study", "You reached the end of this line." %>";
   i18n.translation_chapter = "<%= dgettext "study", "Chapter" %>";
+  i18n.suggestion_first_pgn_arrow = "<%= raw dgettext "study", "The transparent arrows are arrows stored in the study. These don't indicate playable moves but are commonly used to indicate threats, alternative lines, and future ideas." %>";
+  i18n.suggestion_first_neglected_move = "<%= dgettext "study", "One of the moves in the position has been played more than others and is marked with a thinner outline. To give all lines equal attention, play one of the other moves!" %>";
   i18n.suggestion_favorite = "<%= dgettext "study", "If you like this study make sure to favorite it to have it listed under Your Studies" %>";
   i18n.suggestion_share = "<%= dgettext "study", "Do you like this study? Share it with your friends." %>";
   i18n.suggestion_account = "<%= dgettext "study", "Create an account to get all the features of Listudy and upload your own studies." %>";
   i18n.suggestion_comment = "<%= dgettext "study", "Do you have feedback, suggestions or do you want to say something nice? Comment on the study below." %>";
   i18n.suggestion_100moves = "<%= dgettext "study", "You learned 100 moves today. Maybe take a break or come back tomorrow to get the full training effect." %>";
   i18n.suggestion_250moves = "<%= raw dgettext "study", "You learned 250 moves today. For the best training effect come back tomorrow. Or don't, I'm just text not a cop." %>";
-  i18n.arrows_new2x = "<%= dgettext "study", "Arrows: until played 2x" %>";
-  i18n.arrows_new5x = "<%= dgettext "study", "Arrows: until played 5x" %>";
-  i18n.arrows_always = "<%= dgettext "study", "Arrows: always on" %>";
-  i18n.arrows_hidden = "<%= dgettext "study", "Arrows: hidden" %>";
+  i18n.arrows_new2x = "<%= dgettext "study", "Hints: until played 2x" %>";
+  i18n.arrows_new5x = "<%= dgettext "study", "Hints: until played 5x" %>";
+  i18n.arrows_always = "<%= dgettext "study", "Hints: always on" %>";
+  i18n.arrows_hidden = "<%= dgettext "study", "Hints: hidden" %>";
+  i18n.arrow_type_both = "<%= raw dgettext "study", "Hint type: playable & pgn" %>";
```

Comment:
As you say in the video, it would probably be better to replace "pgn". Would "informational" fit instead of "study"? I think that would make it even more obvious.

Reply:
I've been playing around with the word "annotations" in my head. That term would also very easily encompass the circles. It's a little long though. "Study arrows" maybe.. but then that misrepresents the circles and "Hint type: playable & study arrows" is a bit long, but dropping the "arrows" gets a bit cryptic: "Hint type: playable & study". Or maybe "Info arrows". 

Some ideas:

Hint type: playable & annotations
Hint type: playable & info
Hint type: playable & study
Hint type: playable & study arrows
Hint type: moves & annotations
Hint type: moves & study arrows

Reply:
I haven't changed anything regarding this. Looking to hear your opinion first.

My personal favorite I think is:   Hint type: moves & annotations

Reply:
Another option is to remove the word "type" to gain some space.

Reply:
**You could use "informational" but you have to make all options a little longer, and slide the 3rd column a little to the right, as well as remove the word "type" from "Hint type" or it just won't fit.**
![image](https://user-images.githubusercontent.com/41130048/209317494-5a953937-e92d-4333-aa08-b03029cded53.png)

**Here is "Type:" in combination with "annotations", which I personally think is a better word, and also slightly shorter, not requiring any changes to the options widths:**
![image](https://user-images.githubusercontent.com/41130048/209318562-eb92b72b-1c43-48ec-99bf-96b9f865abf0.png)

**Or changing "informational" to just "info" but I think that is less clear.**
![image](https://user-images.githubusercontent.com/41130048/209315932-32b4ec2e-5a7e-4294-8b9b-4ef51720277f.png)

**Here is "moves & annotations", which I think is my favorites:**
![image](https://user-images.githubusercontent.com/41130048/209316042-85779e4c-2ad3-49fe-9149-b75a51e5ba59.png)

**Or another option that can be combined with any of the above is changing "Hint type: " to "Show: ", which I also like:**
![image](https://user-images.githubusercontent.com/41130048/209315957-68333569-2626-42dc-acf7-8619af6194d9.png)

Reply:
After thinking about it for some time I pushed a change that changed the wording to this:

Hints: moves & annotations
Hints: playable moves only
Hints: annotations only

Still, if you prefer something else of course I'll change it.

Also, changing the PR type from Draft to a real PR.